kibbeh/src/ui/UserAvatar/SingleUser.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+
+type size = "default" | "sm" | "xs";
+
+export interface AvatarProps {
+  src: string;
+  size?: size;
+  className?: string;
+  isOnline?: boolean;
+}
+
+const avatarSizeMap = {
+  default: "80px",
+  sm: "40px",
+  xs: "20px",
+};
+
+const onlineIndicatorStyleMap = {
+  default: {
+    width: "15px",
+    height: "15px",
+    right: "6px",
+    bottom: "-9px",
+    borderWidth: "4px",
+  },
+  sm: {
+    width: "8px",
+    height: "8px",
+    right: "2px",
+    bottom: "-4px",
+    borderWidth: "2px",
```

Comment:
Mid size: The red dot (ignoring `border-width`) is `right: 4px` and `bottom: 0`. So taking in account the `border-width`, it should be `right: 2px` and `bottom: -2px`.

Reply:
Done